templates/content/node--oe-person--oe-compact-teaser.html.twig

```diff
@@ -0,0 +1,15 @@
+{#
+/**
+ * @file
+ * Theme override to display a Person node in the Openeuropa Compact Teaser view mode.
+ */
+#}
+<article class="ecl-u-d-flex ecl-u-pv-m ecl-u-border-bottom ecl-u-border-color-grey-15">
+  <div role="img" class="ecl-u-flex-shrink-0 ecl-u-mr-s ecl-u-media-a-s ecl-u-media-bg-size-contain ecl-u-media-bg-repeat-none " style="background-image:url({{ content.portrait_image }})"></div>
+  <div class="ecl-u-flex-grow-1">
+    {% if content.oe_person_jobs|render is not empty %}
```

Comment:
When checking an entity reference field we can just check if the field is empty or not, not if it doesn't render anything. Checking the rendered output is not as reliable as checking the actual entity field. The node object is available in a view mode template:

![image](https://user-images.githubusercontent.com/153362/117626780-bfade000-b177-11eb-92c3-4ad3b0efcc89.png)

Simply do:

```suggestion
    {% if not node.oe_person_jobs.isEmpty() %}
```